osbenchmark/benchmark.py

```diff
@@ -581,9 +582,15 @@ def add_workload_source(subparser):
         action="store_true")
     test_execution_parser.add_argument(
         "--enable-worker-coordinator-profiling",
-        help="Enables a profiler for analyzing the performance of calls in Benchmark's worker coordinator (default: false).",
+        help="Enables a profiler for analyzing the performance of calls in OSB's worker coordinator (default: false). "
+            "Outputs to ~/.benchmark/logs/ as profile.log",
         default=False,
         action="store_true")
+    test_execution_parser.add_argument(
+        "--profiling-sort-type",
+        help=f"Sort profile.log by sort types (column names). Only applies if --enable-worker-coordinator-profiling is provided. "
+            f"Available sort types: {worker_coordinator.AsyncProfiler.SORT_TYPES}. Default is None.",
+        default=None)
```

Comment:
May help to add a check to warn if this argument is used but profiling is not enabled.
Also, does it make sense to provide an option for sort order (asc/desc)?
Finally, it may be useful to be able to specify only specific tasks for profiling.